repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/entity/ComplexFieldEntity.java

```diff
@@ -23,7 +23,6 @@
     @Id
     @Column(name = "id")
     @GeneratedValue(strategy = SEQUENCE, generator = "complex_field_id_seq")
-    @SequenceGenerator(name = "complex_field_id_seq")
```

Comment:
I don't think this is needed as well **generator = "complex_field_id_seq"**!!!!

Reply:
generator = "complex_field_id_seq" is needed otherwise hibernate will use the default sequence name of "hibernate_sequence" (which doesn't exist)